src/main/java/com/liferay/faces/util/cache/Cache.java

```diff
@@ -62,12 +55,12 @@
 	 * @param  key    The key that the value is mapped to.
 	 * @param  value  The value to be cached.
 	 */
-	public V putIfAbsent(K key, V value);
+	public V putValueIfAbsent(K key, V value);
```

Comment:
@stiemannkj1 I was under the impression during the last voice call that we would settle on simple method names like `getValue`, `putValue`, and `removeValue` -- I can see the argument for `putValueIfAbsent` but I feel that the argument would be stronger if both `putValue` and `putValueIfAbsent` were included in the API. But since only one method is necessary for our simple purposes, I think a plain `putValue` with clear Javadoc would suffice.